cmd/amp/cli/cli_test.go

```diff
@@ -0,0 +1,192 @@
+package cli
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	// Amplifier
+	"github.com/appcelerator/amp/api/rpc/stack"
+	"github.com/appcelerator/amp/api/server"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type TestSpec struct {
```

Comment:
Don't export test types in the cli package. Better yet, rename this package to `cli_test`.